lib/cmd/fhc/target.js

```diff
@@ -137,7 +137,7 @@ function undefinedTargCleanup(cb, tar) {
 }
 
 function pingTarget(targ, callback) {
-  request(targ + "/box/srv/1.1/act/sys/auth/logout", function(err, response, body){
```

Comment:
I don't understand why the logout endpoint was called when checking if a target existed. Odd